misc/python/materialize/cli/ci_logged_errors_detect.py

```diff
@@ -243,9 +243,14 @@ def get_known_issues_from_github() -> list[KnownIssue]:
         for match in matches:
             if matches_apply_to:
                 for match_apply_to in matches_apply_to:
-                    known_issues.append(
-                        KnownIssue(match.strip(), match_apply_to.strip().lower(), issue)
-                    )
+                    try:
+                        known_issues.append(
+                            KnownIssue(
+                                match.strip(), match_apply_to.strip().lower(), issue
+                            )
+                        )
+                    except:
+                        print(f"Invalid regex: {match.strip()}, ignoring")
```

Comment:
Can we make this more prominent than just a normal log line? Adding it as issue to the list?

Reply:
+1

Reply:
Thanks!